config_example.json

```diff
@@ -1211,18 +1211,18 @@
    "apiUrlSecondary": "NON_DEFAULT_HTTP_LINK_TO_EXCHANGE_API",
    "proxyAddress": "",
    "websocketUrl": "NON_DEFAULT_HTTP_LINK_TO_WEBSOCKET_EXCHANGE_API",
-   "availablePairs": "BTC_USD,LTC_USD,ETH_USD,ETC_USD,TUSD_USD,USDT_USD,ZEC_USD,ADA_USD,XLM_USD,ZRX_USD,XRP_USD,BAT_USD,PAX_USD,GUSD_USD,USDC_USD,BCH_USD,BSV_USD,TRX_USD,GRIN_USD,DCR_USD,EOS_USD,BTC_TUSD,BTC_USDT,BTC_PAX,BTC_GUSD,BTC_USDC",
-   "enabledPairs": "BTC_USD",
+   "availablePairs": "BTC-USD,LTC-USD,ETH-USD,ETC-USD,TUSD-USD,USDT-USD,ZEC-USD,ADA-USD,XLM-USD,ZRX-USD,XRP-USD,BAT-USD,PAX-USD,GUSD-USD,USDC-USD,BCH-USD,BSV-USD,TRX-USD,GRIN-USD,DCR-USD,EOS-USD,BTC-TUSD,BTC-USDT,BTC-PAX,BTC-GUSD,BTC-USDC",
+   "enabledPairs": "BTC-USD",
    "baseCurrencies": "USD",
    "assetTypes": "SPOT",
    "supportsAutoPairUpdates": true,
    "configCurrencyPairFormat": {
     "uppercase": true,
-    "delimiter": "_"
+    "delimiter": "-"
```

Comment:
Great care needs to be taken whenever these are adjusted for non-upgraded config people, using config_example will work fine as that is what the bot is expecting, however non upgraded configs won't have these changes and will crash the program. Whenever these are adjusted, there needs to be an upgrade path in Run() to ensure that the old value is checked and replaced accordingly

Reply:
Yeah that's fair. I'll revert this change and rely on the okex/okcoin `Run()` func to convert them and resave locally 🦀